src/integrations/prefect-aws/prefect_aws/observers/ecs.py

```diff
@@ -560,7 +560,9 @@ async def mark_runs_as_crashed(event: dict[str, Any], tags: dict[str, str]):
             )
             await propose_state(
                 client=orchestration_client,
-                state=Crashed(message="No active or succeeded pods found for any job"),
+                state=Crashed(
+                    message="No active or succeeded containers found for any job"
```

Comment:
Maybe it would be best to reuse the same messaging as the log above?
```suggestion
                    message=f"The following containers stopped with a non-zero exit code: {container_indentifiers}"
```